src/style.css

```diff
@@ -1 +1,123 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
According to the checklist, you should not use the `*` selector as it impacts performance. It's better to set styles only for elements that require them.